policyengine_us/parameters/gov/states/ny/tax/income/credits/eitc/match.yaml

```diff
@@ -9,7 +9,9 @@ values:
   2003-01-01: 0.3
 metadata:
   unit: /1
-  label: NY EITC percent
+  label: New York EITC percent
   reference:
     - title: Section 606 Credits against tax (d)
       href: https://www.nysenate.gov/legislation/laws/TAX/606
+    - title: New York Instructions for Form IT-201 Line 65
+      href: https://www.tax.ny.gov/pdf/2023/printable-pdfs/inc/it201i-2023.pdf#page=26
```

Comment:
I dont see the 30% here, should be a different page - please check all references for the actual values